docs/griptape-framework/data/image-generation-engines.md

```diff
@@ -0,0 +1,147 @@
+## Overview
+
+Image generation engines facilitate the use of [image generation drivers](../structures/image-generation-drivers.md) by image generation tasks and tools. Each image generation engine defines a `run` method that accepts the inputs necessary for each image generation mode, combines these inputs with any available rulesets, and provides the request to the configured image generation driver.
```

Comment:
Link to reference docs for `Image Generation Engines`

Reply:
This is the page for Image Generation Engines